WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -333,11 +366,20 @@ private async Task<StatusResponse> FetchStatusAsync()
 	{
 		StatusResponse status = new();
 
+		// Select indexer that's behind the most.
+		var i1 = Global.SegwitTaprootIndexBuilderService;
+		var i2 = Global.TaprootIndexBuilderService;
+		if (i1 is null || i2 is null)
+		{
+			throw new NotSupportedException("This is impossible.");
+		}
+		var indexer = i1.LastFilterBuildTime > i2.LastFilterBuildTime ? i2 : i1;
```

Comment:
Shouldn't we return the oldest one? It can happen that one filter building is going well but the other one is behind 2 days - we won't notice.

Reply:
That's what it does.